src/client/components/AdminPanel/AdminListingsToggle.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ToggleDetails } from '@instructure/ui-toggle-details';
+
+import { BruincastListingsTable } from './BruincastListingsTable';
+import { VideoResListingsTable } from './VideoResListingsTable';
+
+export const AdminListingsToggle = ({ shortname, listings, contentType }) => {
+  AdminListingsToggle.propTypes = {
+    shortname: PropTypes.string,
+    listings: PropTypes.array,
+    contentType: PropTypes.string,
+  };
+
+  let TableType = null;
+  switch (contentType) {
+    case 'bruincast':
+      TableType = BruincastListingsTable;
+      break;
+    case 'videores':
+      TableType = VideoResListingsTable;
+      break;
+    default:
+      TableType = null;
+      break;
+  }
+
+  return (
+    <ToggleDetails
+      id={shortname}
```

Comment:
Considering you previously assigned id to shortname, I'm a little confused of whether you are trying to use class label as id or _id as id.. Either way perhaps just make that clear here?